pkgs/development/compilers/crystal/default.nix

```diff
@@ -2,11 +2,11 @@
 
 stdenv.mkDerivation rec {
   name = "crystal-${version}";
-  version = "0.24.2";
+  version = "0.25.0";
 
   src = fetchurl {
     url = "https://github.com/crystal-lang/crystal/archive/${version}.tar.gz";
-    sha256 = "1l7nrrfgz1yxxjphypwzlxj6dbari20p71zb4l0gix09lmas8l6y";
+    sha256 = "1pnx21ky6cqfyv6df4mmjnyd1yh1bvcqkdzq6f0mk0yrkcl57k3q";
   };
 
   prebuiltName = "crystal-0.24.2-1";
```

Comment:
I think we can update prebuilt binaries to 0.25 now? As it have been released 10 hours ago

Reply:
sounds good.